src/ai/backend/agent/agent.py

```diff
@@ -2275,6 +2275,22 @@ async def load_model_definition(
                 }
                 raw_definition = {"models": [_model]}
 
+            case RuntimeVariant.HUGGINGFACE_TGI:
+                _model = {
+                    "name": "tgi-model",
+                    "model_path": model_folder.kernel_path.as_posix(),
+                    "service": {
+                        "start_command": image_command,
+                        "port": MODEL_SERVICE_RUNTIME_PROFILES[RuntimeVariant.HUGGINGFACE_TGI].port,
+                        "health_check": {
+                            "path": MODEL_SERVICE_RUNTIME_PROFILES[
+                                RuntimeVariant.HUGGINGFACE_TGI
+                            ].health_check_endpoint,
+                        },
+                    },
+                }
```

Comment:
Just for conciseness and **to prevent potential mistakes**, you could refer the matched value as follows.
The same applies to other `case` branches.

```suggestion
            case RuntimeVariant.HUGGINGFACE_TGI:
                _model = {
                    "name": "tgi-model",
                    "model_path": model_folder.kernel_path.as_posix(),
                    "service": {
                        "start_command": image_command,
                        "port": MODEL_SERVICE_RUNTIME_PROFILES[runtime_variant].port,
                        "health_check": {
                            "path": MODEL_SERVICE_RUNTIME_PROFILES[
                                runtime_variant
                            ].health_check_endpoint,
                        },
                    },
                }
```